### DISCUSSION TITLE
Windows version link still points to v1.1.7 instead v1.2.0

### DISCUSSION THREAD

**Discussion by raspstar** [Chia-Network/chia-blockchain#7184] [General]
Hi,

Windows version link still points to v1.1.7 instead v1.2.0 - Can you check & correct that?

Thx & regards...

**Comment by SebMoore** [comment]
You can find the new release in the releases section (https://github.com/Chia-Network/chia-blockchain/releases/tag/1.2.0) but yes in the wiki it's incorrect because they called the installer something different this version, breaking the link.
